e2e/utils/fixtureUrl.js

```diff
@@ -15,6 +15,10 @@ const getUrlFixture = ({
 		url += '&flowcardsHostname=https://bs-local.com';
 	}
 
+	if (process.env.E2E_MODE === 'browserstack-pr' || process.env.E2E_MODE === 'local-pr') {
```

Comment:
💅🏽 

```suggestion
	if (isPRMode()) {
```

and

```js
const isPRMode = () => process.env.E2E_MODE.endsWith('-pr')
```

Reply:
i dont think we will nedd to test local test vs pr flowcards.. if we want to test the pr static we can always clone the branch locally and run them that way